com.unity.shadergraph/Editor/Data/Graphs/GraphData.cs

```diff
@@ -1482,6 +1482,27 @@ public void RemoveGraphInput(ShaderInput input)
             ValidateGraph();
         }
 
+        public void MoveCategory(CategoryData category, int newIndex)
+        {
+            if (newIndex > m_CategoryData.Count || newIndex < 0)
+                throw new ArgumentException("New index is not within categories list.");
+            var currentIndex = m_CategoryData.IndexOf(category);
+            if (currentIndex == -1)
+                throw new ArgumentException("Category is not in graph.");
```

Comment:
Same question here - why not use AssertHelpers and just ignore the action in Release?